src/nidas/dynld/raf/UDPArincSensor.cc

```diff
@@ -139,7 +139,12 @@ void UDPArincSensor::open(int flags)
         }
 
         args[argc] = (char *)0;
-        execvp(args[0], args);
+        ILOG(("forking command %s", args[0]));
+        if (execvp(args[0], args) == -1)
```

Comment:
Kudos for checking the `execvp()` return value.  The error message will be more helpful if it includes the error number (errno) and string.  Often `strerror()` is used for this, but it happens that NIDAS already has that functionality buried in a static method `Exception::errnoToString()`.  Since this is an error, an ILOG (for INFO) is not really appropriate, since it's entirely possible INFO messages are not even being logged.  It would be better to use a higher log level, `ELOG()` (for EMERGENCY) is the most common.  eg:
```
ELOG(("UDPArincSensor: error executing command: ") << args[0] << ": error " << errno
           << ": " << Exception::errnoToString(errno));
```

Reply:
if (_ctrl_pid > 0)
    {
        if (kill(_ctrl_pid, SIGTERM) == -1)
        {
            ELOG(("UDPArincSensor: kill() error: ") << ": error " << errno
                        << " : " << Exception::errnoToString(errno));
        }

        if (waitpid(_ctrl_pid, 0, 0) == -1)
        {   
            ELOG(("UDPArincSensor: waitpid() error: ") << ": error " << errno
                        << " : " << Exception::errnoToString(errno));
        }
    }

Should do anything other than log failure?  I assume we don't really want the dsm process to exit.  And do we wait if the kill failed?